packaging/preupgrade-assistant.spec

```diff
@@ -44,9 +44,9 @@ Requires:       sed findutils bash
 Requires:       rpm-python
 Requires:       redhat-release
 Requires:       yum-utils
-Requires:       openscap%{?_isa} >= 0:1.2.8-1
-Requires:       openscap-engine-sce%{?_isa} >= 0:1.2.8-1
-Requires:       openscap-utils%{?_isa} >= 0:1.2.8-1
+Requires:       openscap%{?_isa} >= 0:1.0.10
+Requires:       openscap-engine-sce%{?_isa} >= 0:1.0.10
```

Comment:
I've tried to run `preupg` on:
- RHEL 6.5 - no go, there's no easy workaround for the missing openscap-engine-sce
- RHEL 6.6 - I did get a traceback:
   ```
  File "/usr/lib/python2.6/site-packages/preupg/utils.py", line 537, in config_has_section
    config = configparser.RawConfigParser(allow_no_value=True)
  TypeError: __init__() got an unexpected keyword argument 'allow_no_value'
   ```
   So I removed the `allow_no_value=True` and then the assessment worked and the old (simple) report was generated. This needs to get fixed.

I suggest that we will no longer state anywhere that the Preupgrade Assistant can run on RHEL 6.5.
a) We can probably have the Lab updated:
  - https://access.redhat.com/labs/rhelupgradehelper/
    Use product Red Hat Customer Portal Labs and component RHEL Upgrade Helper

b) No KB article state that the Preupgrade Assistant can run on RHEL 6.5, so there's no update to any of them needed.
c) Update the Requires in spec to:
   ```
   openscap%{?_isa} >= 0:1.0.8
   openscap-engine-sce%{?_isa} >= 0:1.0.8
   openscap-utils%{?_isa} >= 0:1.0.8
   ```

Reply:
I prefer to move evertyhing around RHEL-6.6 to separate PR because the parser returns error during parsing becuuse of lines where only values are used. It's interesting as it maby points to another error in PA when the config has been "sucessfully" parsed on RHEL-6.6. Probably some catch of the exception?

Reply:
Ok. I created an issue for that: https://github.com/upgrades-migrations/preupgrade-assistant/issues/322.